package.json

```diff
@@ -65,6 +65,14 @@
       "description": "Have eslint attempt to fix some errors automatically when saving the file.",
       "type": "boolean",
       "default": false
+    },
+    "scopes": {
+      "title": "Language scopes to use linter",
+      "type": "array",
+      "default": ["source.js", "source.jsx", "source.js.jsx", "source.babel", "source.js-semantic"],
```

Comment:
Separate these on to individual lines (`npm install --save <package>" will re-write the file like this, meaning the next GreenKeeper PR will do so, might as well get it right from the start).